src/modules/material-requests/dto/material-requests.dto.ts

```diff
@@ -132,14 +133,14 @@ export class CreateMaterialRequestDto {
 	type = undefined;
 
 	@IsString()
-	@IsNotEmpty()
+	@IsOptional()
 	@ApiProperty({
 		type: String,
 		description: "The reason for this user's material request",
 		example:
 			'I would like to do research on evolution of the Dutch language in the vrt news across the decades.',
 	})
-	reason: string;
+	reason?: string = '';
 
 	@IsString()
 	@IsEnum(MaterialRequestRequesterCapacity)
```

Comment:
We should delete the homemade enum and replace it with the database generated enum everywhere
MaterialRequestRequesterCapacity => Lookup_App_Material_Request_Requester_Capacity_Enum